types/index.d.ts

```diff
@@ -1,7 +1,7 @@
 // TypeScript Version: 2.8
 
 import * as React from 'react';
-import { Text, GestureResponderEvent } from 'react-native';
+import { Text, GestureResponderEvent, FlatList } from 'react-native';
```

Comment:
how do we feel about alphabetizing stuff?
```suggestion
import { FlatList, GestureResponderEvent, Text } from 'react-native';
```